msal_extensions/persistence.py

```diff
@@ -115,13 +155,28 @@ def __init__(self, location, entropy=''):
 
     def save(self, content):
         # type: (str) -> None
+        data = self._dp_agent.protect(content)
```

Comment:
Can you remind me why we need this change ?

Reply:
Not strictly needed. We were just splitting old line 119 into new line 158 and 160.